### PR TITLE
DOC: document -dev syntactic sugar

### LINKED ISSUE #177: Document *-dev support
**Describe the bug**

Using e.g. `3.10-dev` works, but it's not documented anywhere so it's a secret that can only be discovered by reading the source.

**Which version of the action are you using?**

latest

**Environment**
n/a

**Python Versions**
n/a

**To Reproduce**
n/a

**Run/Repo Url**
n/a

**Screenshots**
n/a

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by neutrinoceros**
**Description:**
Add missing documentation on how to install alpha and beta releases

**Related issue:**
fix #177

**Check list:**
- [ ] Mark if documentation changes are required.
- [ ] Mark if tests were added or updated to cover the changes.

### CODE REVIEW

**Comment by nedbat** [comment]
Yes please :)   I thought maybe "3.11-dev" would get me the current tip of main.  Oh well... :(

**Comment by EwoutH** [comment]
It would be nice to get this into the documentation!

**Comment by neutrinoceros** [comment]
Hi @vsafonkin and @marko-zivic-93, could anyone have a look please ?

**Comment by vsafonkin** [comment]
Hi @neutrinoceros, we will take a look at this, thank you for your contribution.

**Review by vsafonkin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -150,6 +150,7 @@ Check out our detailed guide on using [Python with GitHub Actions](https://help.
     - For every minor version of Python, expect only the latest patch to be preinstalled.
     - If `3.8.1` is installed for example, and `3.8.2` is released, expect `3.8.1` to be removed and replaced by `3.8.2` in the tools cache.
     - If the exact patch version doesn't matter to you, specifying just the major and minor version will get you the latest preinstalled patch version. In the previous example, the version spec `3.8` will use the `3.8.2` Python version found in the cache.
+    - Use `-dev` instead of a patch number (e.g., `3.11-dev`) to install the latest release of a minor version, *alpha and beta releases included*.
```
Comment (hugovk) [inline README.md:153]: "*alpha, beta and release candidates included*" ?

Comment (neutrinoceros) [reply README.md:153]: Yup that would be more accurate, sorry I missed that

**Pull request merged**